styles/style.css

```diff
@@ -29,103 +30,354 @@ body {
-  margin-right: 1rem;
+header nav ul li .header__logo:hover {
+  color: white;
+  background-color: var(--primary-color);
 }
 
-main {
-  background-image: url(../img/header-illsutration-mobile.svg);
-  background-repeat: no-repeat;
-  background-position: top center;
+header nav ul li .header__logo:active {
+  color: white;
+  background-color: var(--secundary-color);
+}
+
+header nav ul li .header__logo:disabled,
+header nav ul li .header__logo[disabled] {
+  color: var(--neutral-color);
+  background-color: white;
+}
+
+header .nav__button {
+  width: 40px;
+  height: 40px;
+  margin-right: 3rem;
   display: flex;
-  flex-direction: column;
+  justify-content: center;
   align-items: center;
-  width: 100%;
-  height: 100%;
+  color: var(--primary-color);
+  background-color: white;
+  border-style: none;
+  text-decoration: none;
 }
 
-.headline {
+header .nav__button i {
+  font-size: 2rem;
+}
+
+header .nav__button:hover {
+  color: white;
+  background-color: var(--primary-color);
+}
+
+header .nav__button:active {
+  color: white;
+  background-color: var(--secundary-color);
+}
+
+header .nav__button:disabled,
+header .nav__button[disabled] {
+  color: var(--neutral-color);
+  background-color: white;
+}
+
+main {
+  width: 100%;
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
+  background-image: url(../img/bg-headline.png);
+  background-repeat: no-repeat;
+  background-position: top center;
 }
 
-.title {
-  width: 32.7rem;
-  margin-top: 6rem;
-  text-align: center;
+main .headline {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  width: 100%;
+  margin-left: 6rem;
 }
 
-.primary {
+h1 {
+  width: 32.7rem;
+  margin-top: 10rem;
   font-family: var(--font-sans);
   font-style: normal;
   font-weight: bold;
   font-size: 4rem;
   line-height: 5.2rem;
-  text-align: center;
+  color: var(--neutral-color);
 }
 
-.title h1 {
-  color: var(--primary-blue);
+p {
+  color: var(--neutral-color);
+  width: 30rem;
+  margin-top: 1.2rem;
+  margin-right: 2.4rem;
+  font-family: var(--font-sans);
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1.5rem;
+  line-height: 2.4rem;
 }
 
-.title h2 {
-  color: var(--primary-green);
+span {
+  font-family: var(--font-sans);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: var(--primary-color);
+  margin-top: 1.2rem;
 }
 
-.supporting-text {
-  text-align: center;
+.headline__social-media {
   color: var(--neutral-color);
-  width: 85%;
-  margin: 4rem;
+  margin-top: 1.2rem;
+}
+
+.social-media__ul {
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+}
+
+.headline__social-media a {
+  margin-right: 2rem;
+}
+
+/* works styles */
+
+.works {
+  width: 100%;
+  display: grid;
+  grid-template-columns: auto;
+  grid-row-gap: 10rem;
+  justify-content: start;
+  margin: 18.4rem 0 8rem 3rem;
+}
```

Comment:
- [ ] Your works section matches the Figma perfectly. you've styled it really well 👏. I noticed your projects section is aligned to the left. Figma's design is aligned in the middle. If you could please make a slight change to the alignment in order to match the Figma template perfectly 👌.
## Your Design
![image](https://user-images.githubusercontent.com/18730106/143718431-5475bd1d-ff78-417f-8805-2e925f9ac4cd.png)
## Figma Design
![image](https://user-images.githubusercontent.com/18730106/143718445-d948b36a-00d9-4a69-af0d-14214fa3390b.png)